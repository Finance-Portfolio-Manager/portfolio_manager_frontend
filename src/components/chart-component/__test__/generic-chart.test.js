import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import GenericChart from './../generic-chart';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = shallow(<GenericChart></GenericChart>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<GenericChart></GenericChart>).toJSON();
    expect(tree).toMatchSnapshot();
})

// MAYBE come back to this to render dummy chart with props
// const wrapper = mount(<GenericChart></GenericChart>);