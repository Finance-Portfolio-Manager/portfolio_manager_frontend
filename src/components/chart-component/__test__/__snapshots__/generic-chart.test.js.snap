// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="example"
>
  <div
    className="row"
  >
    <div
      className="mixed-chart"
    >
      <div
        height="auto"
        options={
          Object {
            "chart": Object {
              "height": 350,
            },
            "title": Object {
              "align": "left",
              "text": undefined,
            },
            "xaxis": Object {
              "convertedCatToNumeric": false,
              "type": "datetime",
            },
            "yaxis": Object {
              "tooltip": Object {
                "enabled": true,
              },
            },
          }
        }
        series={
          Array [
            Object {
              "data": undefined,
            },
          ]
        }
        type="line"
        width="100%"
      />
    </div>
  </div>
</div>
`;
